apps/pdb/src/main.cpp

```diff
@@ -9,16 +9,15 @@ AnalogIn railBattery(RAIL_BATTERY_ANLG_IN), rail5V(RAIL_5V_ANLG_IN), rail17V(RAI
     rail24V(RAIL_24V_ANLG_IN);  // add voltage range (as percentage) to hw bridge also allocate can id for reporting if
                                 // outside range
 
-CANBus can1(CAN1_RX, CAN1_TX, HWBRIDGE::ROVERCONFIG::ROVER_CANBUS_FREQUENCY);
+CANBus can1(CAN1_RX, CAN1_TX, HWBRIDGE::ROVER_CANBUS_FREQUENCY_HZ);
```

Comment:
PDB is still using the old CAN lib, will update it to the new CAN lib in the PDB app PR

Reply:
this is also global, when it should be local imo